testnet.yaml

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
This file shouldn't be included. Add its name to your `.git/info/exclude` so it will be ignored by git.